### PR TITLE
[AAE-16206] security code scan fixes

### PR DESCRIPTION
**Pull request by wojciech-piotrowiak**
Security changes suggested by CodeQL

### CODE REVIEW

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/converter/ProcessModelContentConverter.java [hunk 1]
```diff
@@ -78,7 +79,11 @@ public Optional<BpmnProcessModelContent> convertToModelContent(BpmnModel bpmnMod
 
     public BpmnModel convertToBpmnModel(byte[] modelContent) {
         try (InputStreamReader reader = new InputStreamReader(new ByteArrayInputStream(modelContent))) {
-            XMLStreamReader xmlReader = createSafeXmlInputFactory().createXMLStreamReader(reader);
+            XMLInputFactory safeXmlInputFactory = createSafeXmlInputFactory();
```
Comment (wojciech-piotrowiak) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/converter/ProcessModelContentConverter.java:80]: createSafeXmlInputFactory() method is defined in other module, not sure if CodeQL is able to check code there. Also CodeQL recommend other solution, but result should be the same.
We can use their idea or mark this issue as ignored.
What is weird that by default strict xml looks disabled: https://www.activiti.org/userguide/index.html#advanced.safe.bpmn.xml (but I could not find any control of that logic)

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/converter/ProcessModelContentConverter.java [hunk 2]
```diff
@@ -78,7 +79,9 @@
...
+            safeXmlInputFactory.setProperty("http://apache.org/xml/features/disallow-doctype-decl", true);
+            XMLStreamReader xmlReader = safeXmlInputFactory.createXMLStreamReader(reader);
```
[the remark this answers is not in the document]
Comment (wojciech-piotrowiak) [reply activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/converter/ProcessModelContentConverter.java:81]: Not sure if createSafeXmlInputFactory() is used elsewhere and if CodeQL is able to detect what is inside. 
Open to add fix there if it makes sense

File: activiti-cloud-notifications-graphql-service/services/graphiql/src/main/resources/static/graphiql/graphiql.js [hunk 3]
```diff
@@ -12282,7 +12282,7 @@ function extractLineClasses(type, output) {
     var prop = lineClass[1] ? "bgClass" : "textClass";
     if (output[prop] == null)
       { output[prop] = lineClass[2]; }
-    else if (!(new RegExp("(?:^|\s)" + lineClass[2] + "(?:$|\s)")).test(output[prop]))
```
Comment (mauriziovitale) [inline activiti-cloud-notifications-graphql-service/services/graphiql/src/main/resources/static/graphiql/graphiql.js:12285]: is this \s change intended ?

Comment (wojciech-piotrowiak) [reply activiti-cloud-notifications-graphql-service/services/graphiql/src/main/resources/static/graphiql/graphiql.js:12285]: yes, spotted by code ql https://github.com/Activiti/activiti-cloud/security/code-scanning/2

**Review by mauriziovitale — APPROVED** [review]

File: activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ConnectorValidationControllerIT.java [hunk 4]
```diff
@@ -65,11 +67,14 @@ public class ConnectorValidationControllerIT {
     @MockBean
     private SecurityManager securityManager;
 
+    private MockMvc mockMvc;
+
     @BeforeEach
     public void setUp() {
         webAppContextSetup(context);
```
Comment (Giovanni007) [inline activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-rest/src/test/java/org/activiti/cloud/services/modeling/rest/controller/ConnectorValidationControllerIT.java]: is ` webAppContextSetup(context);` this still needed after the introduction of `mockMvc = MockMvcBuilders.webAppContextSetup(context).build();`?

**Review by Giovanni007 — APPROVED** [review]

**Pull request merged**